### DISCUSSION TITLE
Contact medium

### DISCUSSION THREAD

**Discussion by Febl** [axaq/traviso.js#22] [General]
Hello,
For those of us who are not using twitter, how should we get in touch?
Any "traviso@<your_fav_domain_here>.com email ? or even a contact form on the website - I cannot find anything.

Cheers!
Kind Regards

**Comment by axaq** [comment]
Hi @Febl ,

Unfortunately, I don't have a Traviso email but you can alternatively use the comment section in [Traviso blog](https://www.travisojs.com/blog/tutorial/2015/03/15/basic-isometric-world.html) to ask any questions related to Traviso.

Thanks

**Comment by Febl** [comment]
Hi Hakan (@axaq),
Thank you for getting back to me.
This is unfortunate - I think it would be much better to have a generic email for the project so you may discuss information that are not relevant to a board of issues (like this one) and which I do not feel belong to a comment section either.
I need to spend time around traviso to understand how much of my need is met and how this would work for me. Ultimately I was going to ask if you have a developer setup / prefered IDE and a bunch of other questions around the project and how you see where it would be going.

Kind Regards

**Comment by axaq** [comment]
Hi again @Febl ,

There is no wrong questions here. As long as they are related to Traviso, I am happy to answer them either here or in the comments section of the blog. It also helps me not answer the same questions repeatedly and rather reference through these mediums. 
Regarding your question, I don't have any preferred dev. setup or IDE really. I use VS Code like most JS coders but it is by no means a necessity.
Hope this helps.

**Comment by axaq** [comment]
Hi @Febl ,

After some more digging I have found and activated the "Discussions" tab for this repository. Now, I believe, this is a much better place for Q/A's and general discussions and idea sharing. Hopefully this will help you too. I am now moving this thread from "Issues" to "Discussions".

**Comment by Febl** [comment]
Hi @axaq 
Thank you ! this is I think the best outcome.
